scripts/bundle-image-pickup.sh

```diff
@@ -0,0 +1,187 @@
+    echo -e "==============================================================================" \
+    "\nPOSTGRES:\t\t${POSTGRES_IMAGE:-$DEFAULT_POSTGRES_IMAGE}" \
+    "\nSEARCH_API:\t\t${API_IMAGE:-$DEFAULT_API_IMAGE}" \
+    "\nSEARCH_COLLECTOR:\t${COLLECTER_IMAGE:-$DEFAULT_COLLECTOR_IMAGE}" \
+    "\nSEARCH_INDEXER:\t\t${INDEXER_IMAGE:-$DEFAULT_INDEXER_IMAGE}" \
+    "\nSEARCH_OPERATOR:\t${OPERATOR_IMAGE:-$DEFAULT_OPERATOR_IMAGE}" \
+    "\n==============================================================================\n"
+}
+
+get_default_images_from_csv () {
+  log_color purple "Fetching default component images from ${OPERATOR_CSV_FILEPATH}\n"
+
+  for IMG in $(yq e $OPERATOR_IMAGE_PATH $OPERATOR_CSV_FILEPATH); do
+    if [[ $IMG =~ .*"search-v2-operator".* ]]; then
+      DEFAULT_OPERATOR_IMAGE=$IMG
+    fi
+  done
+
+  for IMG in $(yq e $OPERATOR_ENV_PATH $OPERATOR_CSV_FILEPATH); do
+    if [[ $IMG =~ .*"postgres".* ]]; then
+      DEFAULT_POSTGRES_IMAGE=$IMG
+
+    elif [[ $IMG =~ .*"search-collector".* ]]; then
+      DEFAULT_COLLECTOR_IMAGE=$IMG
+
+    elif [[ $IMG =~ .*"search-indexer".* ]]; then
+      DEFAULT_INDEXER_IMAGE=$IMG
+
+    elif [[ $IMG =~ .*"search-v2-api".* ]]; then
+      DEFAULT_API_IMAGE=$IMG
+    fi
+  done
+
+  display_component_images
+}
+
+update_images_csv () {
+  log_color purple "Preparing to update component: $1 => $2\n"
+
+  # TODO: Replace yq path with $OPERATOR_ENV_PATH. (Note: Adding the env variable seems to cause yq to return no results)
+  if [[ $1 =~ .*"search-indexer".* ]]; then
+    INDEXER_IMAGE=$2
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[1].value = "'$INDEXER_IMAGE'"' $OPERATOR_CSV_FILEPATH
+
+  elif [[ $1 =~ .*"search-collector".* ]]; then
+    COLLECTER_IMAGE=$2
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[2].value = "'$COLLECTER_IMAGE'"' $OPERATOR_CSV_FILEPATH
+
+  elif [[ $1 =~ .*"search-v2-api".* ]]; then
+    API_IMAGE=$2
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[3].value = "'$API_IMAGE'"' $OPERATOR_CSV_FILEPATH
+    
+  elif [[ $1 =~ .*"search-v2-operator".* ]]; then
+    OPERATOR_IMAGE=$2
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].image = "'$OPERATOR_IMAGE'"' $OPERATOR_CSV_FILEPATH
+  fi 
+}
+
+log_color "cyan" "Initializing search bundle image pickup..."
+echo -e "Current dir: $(pwd)\n"
+
+# Create an array containing the Search components that we will focus on for image versioning
+SEARCH_COMPONENTS=(postgresql-13 search-collector search-indexer search-v2-api search-v2-operator)
+get_default_images_from_csv
+
+# Clone the pipeline repository (We need to fetch the latest manifest file to capture the latest builds)
+if [ ! -d $PIPELINE_REPO ]; then
+    log_color "yellow" "Preparing to clone $ORG/$PIPELINE_REPO repository...\n"
+    git clone -b "$RELEASE_BRANCH" git@github.com:$ORG/$PIPELINE_REPO.git
+    echo -e
+fi
```

Comment:
Don't need to clone the entire repo, you could just curl the file that we need.
```
curl GET https://raw.githubusercontent.com/stolostron/pipeline/2.7-integration/manifest.json -H "Authorization: token $GITHUB_TOKEN"
```

Reply:
Updated the script to use curl.